README.md

```diff
@@ -1,239 +1,99 @@
-# Secure Credential Store Plug-in for Zowe CLI
+# Kubernetes Secrets for Zowe CLI & Zowe Explorer
 
-<img src="imgs/Zowe_ConformanceBadge_cli.png" width=25% alt="Zowe Conformance Badge"/>
+The Kubernetes Secrets plugin for Zowe CLI and Zowe Explorer allows you to store your Zowe profile credetials as Kubernetes secrets in your Kubernetes cluster of choice. This is by changing the default credential manager upon installation into Zowe CLI or Zowe Explorer.
```

Comment:
please hyphenate all uses of the word **plug-in** so its consistent w/ Zowe Docs